app-wiki/tiddlers/app/top-level-pages/Topics.tid

```diff
@@ -3,7 +3,7 @@ tags: $:/tags/Page
 
 Number of <a href={{{ [[./pages/topics]addprefix<tv-path-to-root>] }}}>topics</a>: <$count filter="[tag[$:/tags/Link]tags[]] -[[$:/tags/Link]]"/>
 
-<ul class="tc-list-vertical">
```

Comment:
We've already got `tc-list-horizontal`, can't we use that here?

Reply:
IMO it depends, if it is free of unwanted side effects. ... I'll have a look

Reply:
It seems to work, I'll push it.